client/src/Components/SignIn/SignIn.js

```diff
@@ -0,0 +1,55 @@
+import React, {useState} from "react";
+import"./SignIn.css";
+import {Button, Form, FormGroup, Label,Input} from 'reactstrap';
+
+
+const SigIn= () => {
+  const [user, setUser] = useState({
+    email: '',
+    password: ''
+  });
+
+    const handleChange = (e) => {
+      setUser({
+        ...user,
+          [e.target.email]: e.target.value,
+          [e.target.password]: e.target.value
+      });
+    };
+    
```

Comment:
Please just leave 1 empty line between function definitions

Reply:
I deleted it. This peace now is unnecessary.